src/Psalm/Internal/Type/TypeParser.php

```diff
@@ -1411,7 +1411,6 @@ private static function getTypeFromKeyedArrayTree(
                         !is_numeric($property_key)
                         || ($had_optional && !$property_maybe_undefined)
                         || $type === 'array'
-                        || $type === 'callable-array'
```

Comment:
why remove this? TCallableKeyedArray still have a `callable-array` key, no?

Reply:
I have no good reason why it was removed. Now back and tests are unaffected,